src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -44,6 +66,24 @@ class Artwork extends React.Component<Props, any> {
     )
   }
 
+  badges() {
+    const { is_acquireable, is_biddable } = this.props.artwork
+    return (
+      <Badges>
+        {is_acquireable && (
+          <Badge>
+            <BadgeText>BUY NOW</BadgeText>
+          </Badge>
+        )}
+        {is_biddable && (
+          <Badge>
+            <BadgeText>BID</BadgeText>
+          </Badge>
+        )}
+      </Badges>
```

Comment:
Optional: You could not show `<Badges>` if neither exist - just leaves some unused bits in the DOM

Reply:
Oh yea good call!